New pack gen bash script.

JonatanAntoni opened the pull request:
- Add scripts to fetch pre-built libs from Artifactory.
- Add pack build stage to Jenkinsfile.

GuentherMartin reviewed (approved):
I am not able to judge all changes in Jenkinsfile.
CMSIS/RTOS/RTX/LIB/fetch_libs.sh did not work out of the box.

GuentherMartin reviewed (approved):
Looks fine to me.

JonatanAntoni commented:
@RobertRostohar, I have updated the script and checked differences to 5.7.0 release. 
Please find the CI-generated pack at https://jenkins.mcu.aws.arm.com/job/cmsis/job/pre_commit/2/artifact/output/ARM.CMSIS.5.8.0-dirty0.pack

Missing files:
- CMSIS/Include -> Deprecated duplicates, do we still need them?
- CMSIS/SVD/ARM_Example.h -> now kept at CMSIS/Utilities
- CMSIS/SVD/ARM_Example.svd -> now kept at CMSIS/Utilities
- CMSIS/Utilities/CPRJ.xsd -> added

Added folder/files:
- CMSIS/Pack/Bash -> removed
- CMSIS/RTOS/CMSIS_RTOS_Tutorial.pdf -> removed
- CMSIS/Utilities/ARM_Example.h -> repo location of former CMSIS/SVD/ARM_Example.h
- CMSIS/Utilities/ARM_Example.svd -> repo location of former CMSIS/SVD/ARM_Example.svd

I think the pack should be now in good shape.
